--- gdk-pixbuf-0.22.0.org/configure.in	2002-12-19 07:18:18.000000000 +0900
+++ gdk-pixbuf-0.22.0/configure.in	2004-10-03 17:09:36.000000000 +0900
@@ -95,7 +95,7 @@
     	else
     	    exit (1);
     }
-    ], dynworks=true)
+    ], dynworks=true,,dynworks=true)
     LIBS="$oLIBS"
     CFLAGS="$oCFLAGS"
 fi
@@ -116,8 +116,8 @@
 
 if $dynworks; then
    AC_DEFINE(USE_GMODULE)
-   GMODULE_LIBS="`glib-config --libs gmodule`"
-   GMODULE_FLAGS="`glib-config --cflags gmodule`"
+   GMODULE_LIBS="`$bindir/glib-config --libs gmodule`"
+   GMODULE_FLAGS="`$bindir/glib-config --cflags gmodule`"
    AC_SUBST(GMODULE_LIBS)
    AC_SUBST(GMODULE_FLAGS)
    AC_MSG_RESULT(yes)
